morse.rb

```diff
@@ -0,0 +1,51 @@
+@dictionary = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}
+
+def decode_char(letter)
+  @letter = @dictionary[letter]
+  @letter
+end
+
+def decode_word(word)
+  @word_array = word.split
+  @current_char = ''
+  @word_array.each do |one_word|
+    @current_char += decode_char(one_word)
+  end
+  print "#{@current_char} "
```

Comment:
- [ ] We don't need to print the word here, we need to `return` it so it can be used. And we don't need to add a whitespace at the end since it is just one word. You remove the `print` and the whitespace:

  ```
  def decode_word(word)
    ...
    "#{@current_char}"
  end
  ```